### DISCUSSION TITLE
Is there an easy way to duplicate a connection?

### DISCUSSION THREAD

**Discussion by HubKing** [Comfy-Org/ComfyUI#5079] [Q&A]
Let's say you have three nodes horizontally like this

     A       B       C

You have a connection between A's a1 outlet and B's b1 inlet. Let's call this line L1. Now, you want to connect A's a1 to C's c1 inlet to create line L2. You can see the end of L1 that is connected to b1, but not the beginning. To see A, you need scroll horizontally left.

Is there anyway to create L2 without scrolling to the left, something like a modifier + dragging the end of L1 to c1?

**Comment by ltdrdata** [comment]
Please request this feature to the front repo.
https://github.com/Comfy-Org/ComfyUI_frontend
